### ISSUE TITLE
MongoDB needed to run this program

### ISSUE THREAD

**Issue by takeshia** [xphongvn/smart-attendance-system-ta#1]
I would like to use this program, but it seems MongoDB is needed to run this program.
can you include it in the repository?

**Comment by xphongvn** [comment]
That's right! Thank you for interested in our demo. We built this just for fun so we did not have much documentations.

For MongoDB, it's very simple. Just install and create a schema with students ID in them for querying.

**Comment by takeshia** [comment]
ok thank. I will try now.

**Comment by takeshia** [comment]
I have problem running the code. 
Can you tell me which python version, OS and openface lib you have used when developing. 
please see the attached image, this is the error i get.
![image](https://user-images.githubusercontent.com/8132515/61610874-4d63c700-ac95-11e9-86a2-0f26e7ddf209.png)
